platform/security/handler/security-handler-guest/src/main/webapp/css/style.css

```diff
@@ -16,3 +16,43 @@
     margin-left: auto;
     margin-right: auto;
 }
+.app-header {
+    padding: 5px 0px 0px 0px;
+    margin-bottom: 21px;
+    background-color: #ecf0f1;
+    color: #95a5a6;
+    white-space: nowrap;
+    vertical-align: middle;
+    height: 51px;
+}
+
+.app-header > label {
+    width: 33%;
+    overflow: hidden;
+    -ms-text-overflow: ellipsis;
```

Comment:
IE6 supports text-overflow.  IE8 added the prefixed version but you do not need it.

https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow